extensions/aws/tests/MockS3RequestSender.h

```diff
@@ -250,6 +252,10 @@ class MockS3RequestSender : public minifi::aws::s3::S3RequestSender {
     return client_config_;
   }
 
+  bool getUserVirtualAddressing() const {
```

Comment:
This looks like a typo
```suggestion
  bool getUseVirtualAddressing() const {
```

Reply:
Good catch, fixed in 70dd30bb57534a2eb38d3fc5cb76150900a8c076